packages/pages/src/util/runtime.test.ts

```diff
@@ -1,24 +1,31 @@
 /**
  * @jest-environment jsdom
  */
+import * as browserOrNode from "browser-or-node";
 import { getRuntime } from "./runtime.js";
 
+jest.mock("browser-or-node", () => ({
+  __esModule: true,
+  isNode: false,
+  isDeno: true,
+  isBrowser: true,
+}));
+
+declare global {
+  // eslint-disable-next-line no-var
+  var Deno:
```

Comment:
Can this be const?

Reply:
I can't find documentation that says it has to be, but it will cause any reference of `Deno` in the file to show an error https://www.typescriptlang.org/docs/handbook/namespaces.html#ambient-namespaces